src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/IDocumentSnapshot.cs

```diff
@@ -20,9 +20,7 @@ internal interface IDocumentSnapshot
 
     ValueTask<SourceText> GetTextAsync(CancellationToken cancellationToken);
     ValueTask<VersionStamp> GetTextVersionAsync(CancellationToken cancellationToken);
-    ValueTask<RazorCodeDocument> GetGeneratedOutputAsync(
-        bool forceDesignTimeGeneratedOutput,
-        CancellationToken cancellationToken);
+    ValueTask<RazorCodeDocument> GetGeneratedOutputAsync(CancellationToken cancellationToken);
```

Comment:
💡 should we take the time to change this to `GetGeneratedDocumentAsync`? 👀

Reply:
Not yet. I have a whole series of renames coming soon. `IDocumentSnapshot` will become `IRazorDocument`, `IProjectSnapshot` will become `IRazorProject.